quarkus/config-api/src/main/java/org/keycloak/config/DatabaseOptions.java

```diff
@@ -100,10 +100,10 @@ public class DatabaseOptions {
             .defaultValue(10000)
             .build();
 
-    public static final Option<Boolean> DB_ACTIVE_DATASOURCE = new OptionBuilder<>("db-active-<datasource>", Boolean.class)
+    public static final Option<Boolean> DB_ENABLED_DATASOURCE = new OptionBuilder<>("db-enabled-<datasource>", Boolean.class)
             .category(OptionCategory.DATABASE_DATASOURCES)
             .defaultValue(true)
-            .description("Deactivate specific named datasource <datasource>.")
+            .description("Enable specific named datasource <datasource>.")
```

Comment:
Given the default value is `true`, I'd suggest using slightly different desc, maybe something like:
```suggestion
            .description("Disable specific named datasource <datasource>.")
```
WDYT?

Reply:
@vmuzikar Both work for me as we don't have any convention on how to name these boolean prop mappers. 

It depends on the PoV again; it might also be awkward to see "Disable something. Default: true". It might evoke the "disablification" is true, so it is disabled :smile: So, I rather used the "Enable something. Default: true" -> enabled by default. 

Updated a little bit differently - as we usually see for our props.